packages/app/src/sections/leaderboard/Competition.tsx

```diff
@@ -133,91 +137,94 @@ const Competition: FC<CompetitionProps> = ({
 											</StyledOrderType>
 										)
 									},
-									width: compact ? 180 : 120,
+									size: compact ? 180 : 120,
 								},
 								{
-									Header: (
+									header: () => (
 										<TableHeader>{t('leaderboard.leaderboard.table.total-trades')}</TableHeader>
 									),
-									accessor: 'totalTrades',
-									sortType: 'basic',
-									width: compact ? 'auto' : 80,
-									sortable: true,
+									accessorKey: 'totalTrades',
+									sortingFn: 'basic',
+									// width: compact ? 'auto' : 80,
+									size: 80,
+									enableSorting: true,
 								},
 								{
-									Header: (
+									header: () => (
 										<TableHeader>{t('leaderboard.leaderboard.table.liquidations')}</TableHeader>
 									),
-									accessor: 'liquidations',
-									sortType: 'basic',
-									width: compact ? 'auto' : 80,
-									sortable: true,
+									accessorKey: 'liquidations',
+									sortingFn: 'basic',
+									// size: compact ? 'auto' : 80,
+									size: 80,
+									enableSorting: true,
 								},
 								{
-									Header: (
+									header: () => (
 										<TableHeader>{t('leaderboard.leaderboard.table.total-volume')}</TableHeader>
 									),
-									accessor: 'totalVolume',
-									sortType: 'basic',
-									Cell: (cellProps: CellProps<any>) => (
+									accessorKey: 'totalVolume',
+									sortingFn: 'basic',
+									cell: (cellProps) => (
 										<Currency.Price price={cellProps.row.original.totalVolume} />
 									),
-									width: compact ? 'auto' : 100,
-									sortable: true,
+									// width: compact ? 'auto' : 100,
+									size: 100,
+									enableSorting: true,
 								},
 								{
-									Header: <TableHeader>{t('leaderboard.leaderboard.table.pnl')}</TableHeader>,
-									accessor: 'pnlNumber',
-									sortType: 'basic',
-									Cell: (cellProps: CellProps<any>) => (
+									header: () => <TableHeader>{t('leaderboard.leaderboard.table.pnl')}</TableHeader>,
+									accessorKey: 'pnlNumber',
+									sortingFn: 'basic',
+									cell: (cellProps) => (
 										<PnlContainer direction="column">
 											<ColorCodedPrice price={cellProps.row.original.pnl} />
 											<StyledValue>{cellProps.row.original.pnlPct}</StyledValue>
 										</PnlContainer>
 									),
-									width: compact ? 120 : 100,
-									sortable: true,
+									size: compact ? 120 : 100,
+									enableSorting: true,
 								},
 							],
 						},
 					]}
 				/>
 			</DesktopOnlyView>
 			<MobileOrTabletView>
-				{/*@ts-expect-error*/}
 				<StyledTable
 					data={data}
+					// @ts-ignore
```

Comment:
What's the ignore for here?

Reply:
https://github.com/styled-components/styled-components/issues/1803